include/kernel.h

```diff
@@ -1810,7 +2015,10 @@ static inline void *z_impl_k_timer_user_data_get(struct k_timer *timer)
  *
  * @return Current uptime in milliseconds.
  */
-__syscall s64_t k_uptime_get(void);
+static inline s64_t k_uptime_get(void)
```

Comment:
in future that should be changed to u64_t as `k_ticks_to_ms_floor64` returns that anyway.

Reply:
It really should not, but we can have that fight later :)